src/Metadata.elm

```diff
@@ -106,8 +119,10 @@ storyMetadataFromSlug slug contentDict =
         Just content ->
             Just
                 { title = content.title
-                , description = descriptionFromBody content.fullTextMarkdown
+                , description = content.summary
                 , imageSrc = imageSrcFromList content.images
+                , summary = content.summary
```

Comment:
not needed (it confusing in terms of naming but the description is the same

Reply:
I was thinking this and did a bit of work to remove it BUT it is needed because the AC has this one rule that says

*  A story uses the summary text as meta description - with fallback to "A story about [Who]: [Title]" copy TBC

So when a story has the summary field we use just that text.

Otherwise we need to build that string. Stories must have a title so that field is guaranteed but stories don't need an author. So I check if an author is present and basically change how the string is generated, okay, and I am using the translation system so they keep in step with the page language. 

I feel that I could move that logic "higher up" in the pipeline but that would mean passing the language along to that so it can do the translations, which may dirty the separation of concerns.